packages/opentelemetry-plugin-http/package.json

```diff
@@ -39,8 +39,15 @@
     "access": "public"
   },
   "devDependencies": {
+    "@opentelemetry/scope-async-hooks": "^0.0.1",
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.6.9",
+    "@types/semver": "^6.0.1",
+    "@types/shimmer": "^1.0.1",
+    "@types/sinon":"^7.0.13",
+    "sinon": "^7.4.0",
```

Comment:
Looks like latest version is 7.3.2 https://www.npmjs.com/package/sinon

Reply:
![Capture d’écran, le 2019-08-05 à 20 02 53](https://user-images.githubusercontent.com/3801683/62502217-242a5700-b7bc-11e9-9c9a-f74ec11b86e1.png)
weird